soap/src/java/com/zimbra/soap/mail/type/Msg.java

```diff
@@ -164,79 +171,111 @@
     public void setReplyType(String replyType) { this.replyType = replyType; }
+    @GraphQLInputField(name=GqlConstants.IDENTITY_ID, description="The identity referenced by {identity-id} specifies the folder where the sent message is saved.")
     public void setIdentityId(String identityId) {
         this.identityId = identityId;
     }
+    @GraphQLInputField(name=GqlConstants.SUBJECT, description="Subject")
     public void setSubject(String subject) { this.subject = subject; }
+    @GraphQLInputField(name=GqlConstants.HEADERS, description="Headers")
     public void setHeaders(List<Header> headers) { this.headers = headers; }
+    @GraphQLInputField(name=GqlConstants.IN_REPLY_TO, description="Message-ID header for message being replied to")
     public void setInReplyTo(String inReplyTo) { this.inReplyTo = inReplyTo; }
+    @GraphQLInputField(name=GqlConstants.FOLDER_ID, description="Folder ID")
     public void setFolderId(String folderId) { this.folderId = folderId; }
+    @GraphQLInputField(name=GqlConstants.FLAGS, description="Flags")
     public void setFlags(String flags) { this.flags = flags; }
+    @GraphQLInputField(name=GqlConstants.CONTENT, description="Content")
     public void setContent(String content) { this.content = content; }
+    @GraphQLInputField(name=GqlConstants.MIME_PART, description="Mime part information")
     public void setMimePart(MimePartInfo mimePart) { this.mimePart = mimePart; }
+    @GraphQLInputField(name=GqlConstants.ATTACHMENTS, description="Attachments information")
     public void setAttachments(AttachmentsInfo attachments) {
         this.attachments = attachments;
     }
+    @GraphQLInputField(name=GqlConstants.INVITE, description="Invite information")
     public void setInvite(InvitationInfo invite) { this.invite = invite; }
+    @GraphQLInputField(name=GqlConstants.EMAILADDRESSES, description="Email address information")
     public void setEmailAddresses(Iterable <EmailAddrInfo> emailAddresses) {
         this.emailAddresses.clear();
         if (emailAddresses != null) {
             Iterables.addAll(this.emailAddresses,emailAddresses);
         }
     }
-
+    @GraphQLIgnore
     public void addEmailAddress(EmailAddrInfo emailAddress) {
         this.emailAddresses.add(emailAddress);
     }
-
+    @GraphQLInputField(name=GqlConstants.TIMEZONES, description="Timezones")
     public void setTimezones(Iterable <CalTZInfo> timezones) {
         this.timezones.clear();
         if (timezones != null) {
             Iterables.addAll(this.timezones,timezones);
         }
     }
-
+    @GraphQLIgnore
     public void addTimezone(CalTZInfo timezone) {
         this.timezones.add(timezone);
     }
-
+    @GraphQLInputField(name=GqlConstants.FRAGMENT, description="First few bytes of the message (probably between 40 and 100 bytes)")
     public void setFragment(String fragment) { this.fragment = fragment; }
+    @GraphQLIgnore
```

Comment:
can remove input field annotation if want to ignore

Reply:
Fragment for now is fine - we want to ignore extraElements at this time.

Reply:
oops my bad